src/Connection/MysqlConnection.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Mysql\Connection;
+
+use PDO;
+use Yiisoft\Db\Command\Command;
+use Yiisoft\Db\Connection\Connection;
+use Yiisoft\Db\Exception\Exception;
+use Yiisoft\Db\Exception\InvalidConfigException;
+use Yiisoft\Db\Exception\NotSupportedException;
+use Yiisoft\Db\Mysql\Schema\MysqlSchema;
+
+use function constant;
+
+/**
+ * Database connection class prefilled for MYSQL Server.
+ */
+final class MysqlConnection extends Connection
```

Comment:
`Mysql` prefix seems to duplicate information that is already there in namespace.

Reply:
Fixed.